addons/controller_icons/objects/ControllerIconTexture.gd

```diff
@@ -161,13 +169,14 @@ func _reload_resource():
 	emit_changed()
 
 func _load_texture_path_impl():
-	var textures : Array[Texture2D] = []
+	var textures: Array[Texture2D] = []
 	if ControllerIcons.is_node_ready() and _can_be_shown():
 		var input_type = ControllerIcons._last_input_type if force_type == ForceType.NONE else force_type - 1
 		if ControllerIcons.get_path_type(path) == ControllerIcons.PathType.INPUT_ACTION:
 			var event := ControllerIcons.get_matching_event(path, input_type)
 			textures.append_array(ControllerIcons.parse_event_modifiers(event))
-		var tex := ControllerIcons.parse_path(path, input_type)
+		var target_device = force_device if force_device >= 0 else ControllerIcons._last_controller
```

Comment:
Taking into account the previous feedback, don't forget to also offset the enum value by `-1`:
```suggestion
		var target_device = ControllerIcons._last_controller if force_device == ForceDevice.ANY else force_device - 1 
```